wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -116,7 +116,10 @@ <h3 class="usa-accordion__heading">
                                 {{ components.DailyValueTimeSeriesComponent(stations[0].site_no) }}
                             </div>
                         </div>
-                    </div>
+					</div>
```

Comment:
More whitespace issues. We don't use tabs in our code and html gets indented with 4 spaces.